### ISSUE TITLE
Optionally create index page also for eaBrowse

### ISSUE THREAD

**Issue by lgeistlinger** [lgeistlinger/EnrichmentBrowser#14]
From email: @olkap wrote:

Functions ebrowser and eaBrowse do similar things. I would expect that
they generated the output files of similar formats. However, the ebrowser
generates a nice index.html file, and the eaBrowse does not. It would be
more consistent, if both functions generate the same result files.

**Comment by lgeistlinger** [comment]
From email: @lgeistlinger wrote:

`eaBrowse` is intended to report the result of carrying out a single enrichment method.
`ebrowser` is intended to give a comprehensive report of results of DE analysis and putatively the execution of multiple enrichment methods and optionally their combination.
`eaBrowse` is thus a building block of `ebrowser` that is supposed to run fast and to be of lightweight.
As such an index page and the additional cost of building it, is (a) not intended for `eaBrowse`, and (b) not needed, as this can always achieved by executing `ebrowser` with just a single EA method, eg

```
ebrowser(meth="ora", ...)
```

**Comment by lgeistlinger** [comment]
From email: @olkap wrote:

I understand your intention.
However, I cannot use the `ebrowser` because it runs enrichment methods with default settings only. It is not satisiying for me as I would like to obtained adjusted p-values. That is why I used `eaBrowse`.
But, the eaBrowse does not give such nice report, with index file, volcano plot or the full ranking of all the gene sets. That is why I asked for extending the report of `eaBrowse`. If it is not possible, maybe it would be better to allow the user to somehow modify the settings of each algorithm when running the `ebrowser`?

**Comment by lgeistlinger** [comment]
Ok, looking into that. 
I'm going to either optionally add the option to create an index page also with `eaBrowse` and/or allow to pass on specific settings such as p-adjustment from `eBrowser` to `eaBrowse`.

**Comment by lgeistlinger** [comment]
I added the `...` argument to the `ebrowser` function, allowing to pass on additional arguments to customize your enrichment analysis.

This allows to add arguments for `sbea` and `nbea` directly to the `ebrowser` call, for example a multiple testing correction via `padj.method`:

```
ebrowser( meth="ora", perm=0, padj.method="BH",
                 exprs=exprs.file, cdat=cdat.file, rdat=rdat.file, 
                 gs=hsa.gs, org="hsa", nr.show=3)
```

Available now in EnrichmentBrowser v2.10.11 (release) and v2.11.18 (devel).
